src/components/shared/Entity/Create.tsx

```diff
@@ -0,0 +1,214 @@
+                    .doneSubscribing(subscription)
+                    .then(() => {
+                        setFailureState();
+                    })
+                    .catch(() => {});
+            } else {
+                setFailureState();
+            }
+        },
+        doneSubscribing: (subscription: RealtimeSubscription) => {
+            return supabaseClient
+                .removeSubscription(subscription)
+                .then(() => {})
+                .catch(() => {});
+        },
+        exit: () => {
+            resetState();
+
+            navigate(routeDetails.captures.path);
+        },
+        jobFailed: (errorTitle: string) => {
+            setFormState({
+                error: {
+                    title: errorTitle,
+                },
+                status: FormStatus.FAILED,
+            });
+        },
+    };
+
+    usePrompt('confirm.loseData', !exitWhenLogsClose && hasChanges(), () => {
+        resetState();
+    });
+
+    return (
+        <>
+            <LogDialog
+                open={showLogs}
+                token={logToken}
+                title={
+                    <FormattedMessage
+                        id={`${messagePrefix}.save.waitMessage`}
+                    />
+                }
+                actionComponent={logAction}
+            />
+
+            {Header}
+
+            {connectorTagsError ? (
+                <Error error={connectorTagsError} />
+            ) : (
+                <>
+                    <Collapse in={formSubmitError !== null}>
+                        {formSubmitError && (
+                            <EntityError
+                                title={formSubmitError.title}
+                                error={formSubmitError.error}
+                                logToken={logToken}
+                                draftId={draftId}
+                            />
+                        )}
+                    </Collapse>
+
+                    <form id={formID}>
+                        {hasConnectors ? (
+                            <ErrorBoundryWrapper>
+                                <DetailsForm
+                                    connectorTags={connectorTags}
+                                    messagePrefix={messagePrefix}
+                                    accessGrants={combinedGrants}
+                                />
+                            </ErrorBoundryWrapper>
+                        ) : null}
+
+                        {imageTag?.id ? (
+                            <ErrorBoundryWrapper>
+                                <EndpointConfig connectorImage={imageTag.id} />
+                            </ErrorBoundryWrapper>
+                        ) : null}
+
+                        {showCollections && imageTag?.id ? (
+                            <ErrorBoundryWrapper>
+                                <CollectionConfig />
+                            </ErrorBoundryWrapper>
+                        ) : null}
```

Comment:
This will need to be expanded in the future when Captures also has a collections component to render.